packages/ppx_netblob/ppx_netblob.0.1/url

```diff
@@ -0,0 +1,2 @@
+http: "https://github.com/chrismamo1/ppx_netblob/archive/master.tar.gz"
```

Comment:
The URL should point to a tag, not to `master`, e.g.

    https://github.com/chrismamo1/ppx_netblob/archive/v0.1.tar.gz